src/index.js

```diff
@@ -1,3 +1,4 @@
+var get = require('lodash/get')
```

Comment:
I think that is okay, I see other projects does this as well and they should really :) No need to reinvent the wheel